R/helpers.R

```diff
@@ -30,9 +67,13 @@
       indexes <- which(dataset[["name"]] == name)
       index <- .resolveConflicts(name, indexes, dataset)
       if (is.null(index)) {
-        return(NULL)
+        city <- .geocode(
+          name = name$name, country = name$country
+        )
+      } else {
+        city <- dataset[index, ]
```

Comment:
```suggestion
        return(NULL)
```

Reply:
Have restore .getCity to its original state without using the fallback geocoding.